src/site/stages/build/drupal/load-saved-flags.js

```diff
@@ -13,6 +13,7 @@ const fs = require('fs-extra');
  *                   an error if a flag is called without existing
  */
 function useFlags(rawFlags, shouldLog = true) {
+  // eslint-disable-next-line fp/no-proxy
```

Comment:
It's worth noting that this isn't browser code, hence the exception. Node doesn't mind proxies.